vendor/frida/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
-set(FRIDA_LINKER_FLAGS
-  -lresolv -lpthread -ldl
-  CACHE STRING "Frida gum flags")
+if(WIN32)
```

Comment:
`if(NOT WIN32)`

Reply:
Changed.  I amended the commit (and force-pushed).